frontend/awx/access/credentials/hooks/useCredentialActions.tsx

```diff
@@ -8,15 +8,24 @@ import {
   PageActionType,
   usePageNavigate,
 } from '../../../../../framework';
-import { cannotDeleteResource, cannotEditResource } from '../../../../common/utils/RBAChelpers';
+import {
+  cannotDeleteResource,
+  cannotEditResource,
+  cannotCopyResource,
+} from '../../../../common/utils/RBAChelpers';
 import { Credential } from '../../../interfaces/Credential';
 import { AwxRoute } from '../../../main/AwxRoutes';
 import { useDeleteCredentials } from './useDeleteCredentials';
+import { useCopyCredential } from './useCopyCredential';
 
-export function useCredentialActions(options?: { onDeleted: (credentials: Credential[]) => void }) {
+export function useCredentialActions(
+  options?: { onDeleted: (credentials: Credential[]) => void },
+  onCredentialCopied = () => null
+) {
   const { t } = useTranslation();
   const pageNavigate = usePageNavigate();
   const deleteCredentials = useDeleteCredentials(options?.onDeleted);
+  const copyCredential = useCopyCredential(onCredentialCopied);
```

Comment:
We need to refresh the main table once the credential is copied, I was not able to see that one happened.